code/game/machinery/camera/camera.dm

```diff
@@ -72,9 +67,18 @@ CREATION_TEST_IGNORE_SUBTYPES(/obj/machinery/camera)
 
 /obj/machinery/camera/Initialize(mapload, obj/structure/camera_assembly/CA)
 	. = ..()
-	for(var/i in network)
-		network -= i
-		network += LOWER_TEXT(i)
+	var/static/list/autonames_in_areas = list()
+
+	// Calculate area code
+	var/area/camera_area = get_area(src)
+	if (istype(camera_area, /area/space))
+		var/turf/connected_wall = get_step(src, dir)
+		camera_area = get_area(connected_wall)
```

Comment:
I think this will return `null` when a random wall was built on a space turf.

Reply:
It will return /area/space